client/src/routes/index.tsx

```diff
@@ -41,16 +39,27 @@ const containerStyles = 'p-[var(--main-content--padding)]';
 
 const LoggedIn = () => {
   useSetBackgroundColor('#1A101C');
-
   // Use startOfHour to prevent infinite loop with a brand new date each time
   // this component unsuspends
   const timestamp = startOfHour(new Date()).toISOString();
 
-  const { data } = useSuspenseQuery<IndexRouteQuery, IndexRouteQueryVariables>(
-    INDEX_ROUTE_QUERY,
-    { variables: { timestamp } }
+  const [queryRef] = useBackgroundQuery(INDEX_ROUTE_QUERY, {
+    variables: { timestamp },
+  });
+
+  return (
+    <Suspense fallback={<h1 style={{ backgroundColor: 'red' }}>loading</h1>}>
```

Comment:
Rather than using the `<h1 />` here, could we use the [`LoadingState`](https://github.com/apollographql/spotify-showcase/blob/42e7600576db23f98530e07c650ca7e25506d4b5/client/src/routes/index.tsx#L266) component in this file as the Suspense fallback? That should allow us to remove the `<Suspense />` wrapper in the [router](https://github.com/apollographql/spotify-showcase/blob/42e7600576db23f98530e07c650ca7e25506d4b5/client/src/router.tsx#L52).